prow/plugins/trigger/pr.go

```diff
@@ -55,21 +56,21 @@ func handlePR(c client, trigger *plugins.Trigger, pr github.PullRequestEvent) er
 		}
 	case github.PullRequestActionReopened:
 		// When a PR is reopened, check that the user is in the org or that an org
-		// member had said "/ok-to-test" before building.
-		comments, err := c.GitHubClient.ListIssueComments(org, repo, num)
+		// member had said "/ok-to-test" before building, resulting in label ok-to-test.
+		labels, err := c.GitHubClient.GetIssueLabels(org, repo, num)
```

Comment:
In this case you do need to get the labels from the API because `github.PullRequestEvent` does not include the labels on the PR.